app/routes/mandatenbeheer/mandatarissen/edit.js

```diff
@@ -26,6 +26,9 @@ export default class MandatenbeheerMandatarissenEditRoute extends Route {
     return RSVP.hash({
       bestuurseenheid: parentModel.bestuurseenheid,
       bestuursorganen: parentModel.bestuursorganen,
+      // TODO temporary workaround, we will switch to an application layout where only a  single
+      // bestuursorgaan is selected
+      bestuursorgaan: parentModel.bestuursorganen[0],
```

Comment:
I think it is a bit weird to pass bestuurorganen and bestuursorgaan both, but since this will change in the future I can live with it.

Reply:
agreed!